styleguide/source/_patterns/00-atoms/00-intro/00-intro.twig

```diff
@@ -7,7 +7,7 @@
   <ul>
     <li>{% include '00-link.twig' with {'content' : 'Twig' , 'url' : 'http://twig.sensiolabs.org/'} %} - for PHP-based HTML templating.</li>
     <li>{% include '00-link.twig' with {'content' : 'SCSS' , 'url' : 'http://sass-lang.com/documentation/file.SCSS_FOR_SASS_USERS.html'} %} - for making awesome style sheets.</li>
-    <li>{% include '00-link.twig' with {'content' : 'BEM (Block Element Modifier)' , 'url' : 'http://getbem.com/'} %} - a methodology that enables creation of reusable components and code sharing in the front-end.</li>
+    <li>{% include '00-link.twig' with {'content' : 'BEM (Block Element Modifier)' , 'url' : 'http://getbem.com/'} %} - a CSS naming methodology that enables creation of reusable components and code sharing in the front-end. We use the abridged syntax recommended in the {% include '00-link.twig' with {'content': '18F Front End Guide', 'url': 'https://frontend.18f.gov/#bem'} %} under their suggested custom methodology section.</li>
```

Comment:
This link doesn't work. :( Looks like there may be a solution [here, maybe.](https://github.com/pattern-lab/patternlab-php/issues/234)?